lib/driver/sdr_init.c

```diff
@@ -17,11 +17,13 @@ static int sdr_driver_init(sdr_interface_data_t *ifdata, const char *ifname) {
             return rc;
         }
     }
+    #ifndef OS_POSIX
```

Comment:
Oops! I guess sdr_sband_driver_init brought in SPISbandTx - sorry

Reply:
I'm not sure if this is the best way to exclude it though

Reply:
I'm not sure either, but I'm also not sure I should be leaving sdr_sband.c in CSP either, it's just that sdr_uart.c is there

Reply:
Perhaps we should look in to modifying the config file generator and adding stuff to csp_autoconfig.h. Would be better than our esoteric way of doing it with these undocumented compile flags

Reply:
I've also had a problem with waf not propagating OS_POSIX into ex2_sdr. I think it looked like that bit you too, because we have this if-nothing-is-defined-default-to-posix thing